assembl/static2/js/app/components/cookies/cookiesSelector.jsx

```diff
@@ -0,0 +1,96 @@
+// @flow
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Button } from 'react-bootstrap';
+import classnames from 'classnames';
+import CookieToggle from './cookieToggle';
+import type { CookieObject } from './cookieToggle';
+
+type CookiesSelectorProps = {};
+
+type CookiesSelectorState = {
+  activeKey: ?string,
+  show: boolean,
+  cookies: Array<CookieObject>
+}
+
+
+class CookiesSelector extends React.Component<CookiesSelectorProps, CookiesSelectorState> {
+  constructor(props: CookiesSelectorProps) {
+    super(props);
+    const cookiesList = document.cookie.split(' ');
```

Comment:
Are you sure that `document.cookie` may not change during the lifetime of this component?

Reply:
I don't think so, but do you have a specific case in mind?